next.json

```diff
@@ -1050,6 +1050,8 @@
                 "name": "configure",
                 "args": [
                     {"name": "format", "type": "texture format"},
+                    {"name": "allowedUsage", "type": "texture usage bit"},
+                    {"name": "initialUsage", "type": "texture usage bit"},
```

Comment:
@kainino0x: didn't we say that when the next texture is provided it should be in the None usage so we force a transition? This would remove the need for initialUsage.